.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,63 @@
+<!--
+BEFORE YOU CREATE A PULL REQUEST:
+
+Ensure you have read over [CONTRIBUTING.md](./CONTRIBUTING.md). We provide VERY defined guidance (as such, we strongly adhere to it).
+
+A summary of our expectations:
+ - You are not submitting a pull request from your MASTER / MAIN branch.
+ - You are able to sign the Contributor License Agreement (CLA).
+ - YOUR GIT COMMIT MESSAGE FORMAT IS EXTREMELY IMPORTANT. We have a very defined expectation for this format and are sticklers about it. Really, READ the entire Contributing document. It will save you and us pain.
+ - Do not reformat code, it makes it hard to see what has changed. Leave the formatting to us.
+
+THANKS! We appreciate you reading the entire Contributing document and not just scanning through it.
+
+Name your issue appropriately: give it a sentence that reads well enough for anyone seeing this in release notes to know what the issue is.
+
+When writing out the pull request details please ensure you are writing it as
+if you were explaining it to somebody else.
+Even if you will be working on and resolving the issue yourself.
+This helps others to understand the reasons for the pull request and for it to be searchable in future.

```

Comment:
This section looks a bit odd.
The sentence on line `18` that says `Even if you will be working on and resolving the issue yourself.` don't sound appropriate for a pull request, should this have been in the issue template itself?

Reply:
Agreed. Removed from PR template and added to issue one.